src/site/stages/build/drupal/page.js

```diff
@@ -145,9 +145,12 @@ function generateBreadCrumbs(pathString) {
   let previous = '';
   let trimmedValue;
 
-  for (const value of pathArray) {
+  for (let value of pathArray) {
```

Comment:
let's leave this as a constant and break this mutation into smaller transformations. 

you should be able to chain off of the current assignment statement with `.map`: 
```
 const pathArray = pathString
  .split('/')
  .map(value => value === health-services ? 'our-health-services' : value);
```

Optionally we could add additional refactors here to clear up what this is doing and rely less on nested if blocks: 

```
.reduce((breadcumbParts, pathPart) => {
    breadcrumbParts.push(
      { 
      pathPart,
      text: value === 'pittsburgh-health-care'
          ? 'VA Pittsburgh health care'
          : _.startCase(_.trim(value, '-')),
      }
    );
    return breadcrumbParts;
  }
, [])
```

and, maybe something like this (didn't check the logic completely- i'm not sure why this is checking `if (value) {` in the original). Some of the naming might be tweaked (I don't fully understand what's changing here). 

```
.forEach(breadcrumbPart => {
    const { pathPart, text } = breadcrumbPart;
    trimmedValue = _.trim(breadcrumbPart.path, '/');
    entityUrlObj.breadcrumb.push({
        url: {
          path: `${previous}${pathPart}`,
          routed: true,
        },
        text,
      });
    }
    previous += `${trimmedValue}/`;
  }
```

Reply:
:+1: updated, @rianfowler